map/map_loader/src/lanelet2_map_loader/lanelet2_local_projector.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 The Autoware Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LANELET2_MAP_LOADER__LANELET2_LOCAL_PROJECTOR_HPP_
+#define LANELET2_MAP_LOADER__LANELET2_LOCAL_PROJECTOR_HPP_
+
+#include <lanelet2_io/Projection.h>
+
+namespace lanelet::projection
+{
+
+class LocalProjector : public Projector
+{
+public:
+  LocalProjector() : Projector(Origin(GPSPoint{})) {}
+
+  BasicPoint3d forward(const GPSPoint & gps) const override  // NOLINT
+  {
+    return BasicPoint3d{0.0, 0.0, gps.ele};
+  }
+
+  GPSPoint reverse(const BasicPoint3d &) const override { return {}; }
```

Comment:
Since `forward` holds the height of gps, shouldn't `reverse` also hold the height?

Reply:
Thank you. Updated it with https://github.com/autowarefoundation/autoware.universe/pull/5866/commits/cb572e634969c681b52a40d864c3b03d0077b831.